### PR TITLE
Fix rejection line calculation in Benjamini Yekutieli example

### PR DESCRIPTION
**Pull request by alxfmpl**
Running the Benjamini Yekutieli visualisation example, I noticed that the BY rejection line calculation does not yield the expected result. This PR aligns the BY rejection line calculation used in this example with the statsmodels implementation.

The problem is similar to #508.

### CODE REVIEW

File: notebooks/advanced/visualize-benjamini-yekutieli-procedure.ipynb [hunk 1]
```diff
@@ -708,7 +317,7 @@
     "if HYPOTHESES_INDEPENDENT:\n",
     "    C = [1] * m\n",
     "else:\n",
-    "    C = [sum([1.0 / i for i in range(1, k + 1)]) for k in K]\n",
+    "    C = [sum([1.0 / k for k in K])] * m\n",
```
Comment (nils-braun) [inline notebooks/advanced/visualize-benjamini-yekutieli-procedure.ipynb:320]: @MaxBenChrist Do you think you will have some time to have a look into this?

**Review by nils-braun — COMMENTED** [review]
@alxfmpl Thank you very much for your contribution! Very happy to see this.
I have asked @MaxBenChrist to maybe review this PR (as he was, if I remember correctly, the original author if this notebook).

**Comment by nils-braun** [comment]
You can ignore the failing benchmark test

File: notebooks/advanced/visualize-benjamini-yekutieli-procedure.ipynb (hunk 1 above)
Comment (MaxBenChrist) [reply notebooks/advanced/visualize-benjamini-yekutieli-procedure.ipynb:320]: Yes, will try to free some time the next days.

Comment (nils-braun) [reply notebooks/advanced/visualize-benjamini-yekutieli-procedure.ipynb:320]: @MaxBenChrist Gentle reminder :-)

**Comment by alxfmpl** [comment]
Upon review I noticed my problem/fix description was beating around the bush a little 😉. Basically this PR boils down to this:

With `C = [sum([1.0 / i for i in range(1, k + 1)]) for k in K]` the rejection line in the example looks something like this:

grafik

This is not really what I expected. So after consulting [Wikipedia about the BY procedure](https://en.wikipedia.org/wiki/False_discovery_rate#Benjamini%E2%80%93Yekutieli_procedure), the rather [confusing paper introducing the BY procedure](https://projecteuclid.org/download/pdf_1/euclid.aos/1013699998) and the statsmodels implementation, I figured out that with `C = [sum([1.0 / k for k in K])] * m` the rejection line looks a lot more like what I was expecting:

grafik

Hope this helps.

**Comment by kempa-liehr** [comment]
Thanks, for the clarification :-)

**Comment by nils-braun** [comment]
@kempa-liehr You assigned yourself to this. Are you fine with the changes? Shall we merge?

**Comment by MaxBenChrist** [comment]
Better late than never: The changes are correct and we should merge them. 

some additional thoughts
- I would put the calculation of the rejection line inside the notebook into a function
- you stripped the cells of their output, do we want to keep the notebooks with or without output in the repository? the other notebooks in the tsfresh example folder have their output attached

**Review by MaxBenChrist — APPROVED** [review]

**Comment by alxfmpl** [comment]
Hi @MaxBenChrist, thanks for your approval.

Regarding your thoughts:

1.  I'm more than happy to create a function for rejection line calculation if wished.
2. According to the [contributing guidlines](https://github.com/blue-yonder/tsfresh/blob/main/docs/text/how_to_contribute.rst#guidelines) notebook outputs should be removed. This obviously makes the diff really large. I think keeping the old outputs would be a bit misleading through, especially since notebooks including their outputs are rendered on GitHub.

**Comment by nils-braun** [comment]
@alxfmpl Thank you very much for taking the time and implementing the fix and I am very sorry it took so long on our side...

**Comment by alxfmpl** [comment]
@nils-braun No worries, thanks for the merge 👍

**Pull request merged**